websocket/services/auctionRoundNumberService.ts

```diff
@@ -0,0 +1,42 @@
+import { ClickHouseClient } from "@clickhouse/client";
+import { ABLY_CHANNELS } from "@mevscan/shared/ablyConstants";
+import { config, NodeEnv } from "@mevscan/shared/config";
+import Ably from 'ably';
+
+async function getAuctionRoundNumber(clickhouseClient: ClickHouseClient): Promise<number | null> {
+    const query = `
+        SELECT 
+        MAX(round) as auctionRoundNumber 
+        from timeboost.auction
+    `;
+
+    const result = await clickhouseClient.query({
+        query,
+        format: 'JSONEachRow',
+    });
+
+    const data = await result.json<Array<{ auctionRoundNumber: number }>>();
+    return data[0]?.auctionRoundNumber ?? null;
+}
```

Comment:
Lets push a bit more than auction number here. the auction table has many more columns which are pretty useful.